### DISCUSSION TITLE
Look

### DISCUSSION THREAD

**Discussion by MCHSXIII** [vigoren/foundryvtt-simple-calendar#237] [Q&A]
Heya,
I love the simple calendar module but its ui/look is really throwing
me out of immersion.
Is there a way to change it into sth that looks Fantasy/Sci Fi?
(I play both dnd and starwars)
(If there is an obvious way that i didnt see i am really sorry,
i did go through the options before asking here, but theres a lot of stuff going on in the module
and i couldnt find any design settings)

**Comment by vigoren** [comment]
Hi!
There currently isn't any way to change the look of calendar though this is something I'm looking at doing eventually.

That said it's been a low priority because there's are a couple of modules that provide a nicer look. I would check out [SmallTime](https://foundryvtt.com/packages/smalltime) and [Weather Control](https://foundryvtt.com/packages/weather-control) for their interfaces.
